Frends.Excel.ConvertToXML/Frends.Excel.ConvertToXML/Excel.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System.Data;
+using System.Globalization;
+using System.Text;
+using System.Xml;
+using ExcelDataReader;
+
+namespace Frends.Excel.ConvertToXML;
+
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to XML. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.ConvertToXML)
+    /// </summary>
+    /// <param name="input">Input configuration</param>
+    /// <param name="options">Input options</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns>Result containing the converted XML string.</returns>
+    /// <exception cref="Exception"></exception>
+    public static Result ConvertToXML(Input input, Options options, CancellationToken cancellationToken)
+    {
+        try
+        {
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+            using (var stream = new FileStream(input.Path, FileMode.Open))
+            {
+                using (var excelReader = ExcelReaderFactory.CreateReader(stream))
+                {
+                    var result = excelReader.AsDataSet();
+                    var xml = ConvertDataSetToXml(result, options, Path.GetFileName(input.Path), cancellationToken);
+                    return new Result(true, xml, null);
+                }
+            }
+        }
+        catch (Exception ex)
+        {
+            if (options.ThrowErrorOnFailure)
+            {
+                throw new Exception("Error while converting Excel file to XML", ex);
+            }
+
+            return new Result(false, null, $"Error while converting Excel file to XML: {ex}");
+        }
+    }
+
+    /// <summary>
+    /// Converts DataSet-object to XML.
+    /// </summary>
+    /// <param name="result">DataSet-object</param>
+    /// <param name="options">Input configurations</param>
+    /// <param name="file_name">Excel file name to be read</param>
+    /// <param name="cancellationToken">Cancellation token</param>
+    /// <returns>String containing contents in XML format.</returns>
+    public static string ConvertDataSetToXml(DataSet result, Options options, string file_name,
```

Comment:
Helper methods need to be private. If FrendsMetadata.json is not provided, Frends will also try to import this method. Also, maybe add #region HelperMethods to indicate that these methods are helper methods to developer.